src/binary-search-tree.js

```diff
@@ -2,6 +2,26 @@
 /* eslint-disable global-require */
 /* eslint-disable no-unused-vars */
 /* eslint-disable no-trailing-spaces */
+class Queue {
```

Comment:
This class should definitely be imported instead of pasting the code here in this file.